examples/basic/hello-world/README.md

```diff
@@ -0,0 +1,16 @@
+# Cycle.js Hello World example
+
+This is the smallest possible cycle.js program. It just echoes anything written
```

Comment:
Could you say "this is a very small" instead of "smallest possible"?

Also, replace `cycle.js` with `Cycle.js` for consistency.

Reply:
Done.